lib/vfscore/Makefile.uk

```diff
@@ -10,6 +10,7 @@ LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/file.c
 LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/stdio.c
 LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/mount.c
 LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/vnode.c
+LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/volumes.c
```

Comment:
Can we name the file `automount.c`? I think as a little clean-up it makes sense to rename `rootfs.c` and then add your code on top. Then we have it within one file.

Reply:
Are you sure? Because that would make volumes depend on `CONFIG_LIBVFSCORE_AUTOMOUNT_ROOTFS`. And there is a case where you may want to have atuomatic mount of volumes without having the `rootfs.c` functionality.

Reply:
So guard two thirds of the file? Does it not look like bad design since the separation is so big to not have them in separate files so that the separation is transparent from the source code point of view?